cmd/conformance-tester/pkg/scenarios/generator.go

```diff
@@ -184,7 +184,7 @@ func providerScenario(
 		scenario := &vSphereScenario{baseScenario: base}
 		scenario.customFolder = opts.ScenarioOptions.Has("custom-folder")
 		scenario.basePath = opts.ScenarioOptions.Has("basepath")
-		//scenario.datastoreCluster = opts.ScenarioOptions.Has("datastore-cluster")
+		// scenario.datastoreCluster = opts.ScenarioOptions.Has("datastore-cluster")
```

Comment:
You need tot disable the prow job https://github.com/kubermatic/kubermatic/blob/main/.prow/provider-vsphere.yaml#L153